Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationWindow.cs

```diff
@@ -136,13 +159,113 @@ private void OnGUI()
 #endif
         }
 
-        private void DrawRecordingGUI()
+        private void DrawSimulationGUI()
         {
-            if (!Application.isPlaying)
+            if (SimulationService == null)
             {
-                EditorGUILayout.HelpBox("Input test recording is only available in play mode", MessageType.Info);
+                EditorGUILayout.HelpBox("No input simulation service found", MessageType.Info);
                 return;
             }
+
+            DrawHeadGUI();
+            DrawHandsGUI();
+        }
+
+        private void DrawHeadGUI()
+        {
+            if (!CameraCache.Main)
+            {
+                return;
+            }
+
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                GUILayout.Label($"Head:");
+
+                Transform headTransform = CameraCache.Main.transform;
+                Vector3 newPosition = EditorGUILayout.Vector3Field("Position", headTransform.position);
+                Vector3 newRotation = DrawRotationGUI("Rotation", headTransform.rotation.eulerAngles);
+                bool resetHand = GUILayout.Button("Reset");
+
+                if (newPosition != headTransform.position)
+                {
+                    headTransform.position = newPosition;
+                }
+                if (newRotation != headTransform.rotation.eulerAngles)
+                {
+                    headTransform.rotation = Quaternion.Euler(newRotation);
+                }
```

Comment:
Super nit: Move these into an else {} statement of resetHand - no need to set them if we're just gonna reset them to identity.

Reply:
It's just a UI script, the "else" case runs in, like, 99.999% of draw calls :)
No need to make it complicated IMO.